test/cases/linux/ops-agent/1.0.4/custom_config/config.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2020 Google LLC
```

Comment:
Are the test files for `1.0.4` maintained by some automation system (aka auto generated), or entered by hand?

Reply:
They are not generated, created by hand. This is okay for now but more complex CI should probably have some sort of generation process.